lib/asset_sync/storage.rb

```diff
@@ -58,6 +62,32 @@ def local_files
         (get_local_files + config.additional_local_file_paths).uniq
     end
 
+    def remote_files
+      return @remote_files if @remote_files
+      return @remote_files = [] if ignore_existing_remote_files?
```

Comment:
I believe when `ignore_existing_remote_files?` is `true`, `remote_files` should be an empty array no matter the value of `remote_file_list_cache_file_path`.
Tell me if I should do otherwise.

Reply:
Your statement is correct.

But I think the code should be 
```ruby
return [] if ignore_existing_remote_files?
return @remote_files if @remote_files
# ...
@remote_files = get_remote_files
```
Since the original logic is `ignore_existing_remote_files? ? [] : get_remote_files`
which equals
```ruby
return [] if ignore_existing_remote_files?

get_remote_files
```

Reply:
I thought those two are identical in this case, but your suggestion is easier to understand.
Thanks.